hellenistic_cavalry_warfare.md

```diff
@@ -0,0 +1,3 @@
+Cavalry during the Hellenistic period was used in many different forms. Light mounted units were often assigned intelligence-gathering tasks. They also employed hit-and-run tactics to keep enemies busy, exhaust them, or lead them into traps. In battle, light cavalry was usually tasked with securing the infantry's flanks. When attacking, they oftentimes attempted maneuvers of outflanking, encircling, or pulling apart to open a gap in the opposing formation. And because of their speed, light cavalry was also used to pursue and kill off fleeing soldiers. 
```

Comment:
```suggestion
Cavalry during the Hellenistic period was used in many different forms. Light, mounted units were often assigned intelligence-gathering tasks. They also employed hit-and-run tactics to keep enemies busy, exhaust them, or lead them into traps. In battle, light cavalry was usually tasked with securing the infantry's flanks. When attacking, they oftentimes attempted maneuvers of outflanking, encircling, or pulling apart to open a gap in the opposing formation. And because of their speed, light cavalry was also used to pursue and kill off fleeing soldiers. 
```